makerspace/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from web.multilingual.database import MultiLingualTextField, MultiLingualRichTextUploadingField
+
+
+class Tool(models.Model):
+    title = MultiLingualTextField(
+        max_length=100,
+        verbose_name=('Title'),
```

Comment:
The `verbose_name` fields should maybe be translated. This can be done using pythons [gettext_lazy](https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.translation.gettext_lazy).

Reply:
They should, the translations are used in the automatic labels in forms. We usually import `gettext_lazy as _`, so we write `verbose_name=_('Title')`